packages/components/src/components/code-snippet/_code-snippet.scss

```diff
@@ -59,14 +59,6 @@
     padding: 0 $spacing-xs;
   }
 
-  .#{$prefix}--snippet--inline.#{$prefix}--snippet--light {
-    background-color: $field-02;
-
-    &:hover {
-      background-color: rgba($interactive-02, 0.1);
```

Comment:
**Please note:** this style rule isn't valid & currently the `inline` + `light` variation of the `CodeSnippet` doesn't have a background-color on hover as a result: http://react.carbondesignsystem.com/?path=/story/codesnippet--inline

![Screen Shot 2019-10-15 at 2 12 36 PM](https://user-images.githubusercontent.com/9057921/66862289-3cee8000-ef56-11e9-9701-981eb4c18a5b.png)

Reply:
I believe this is valid Sass but these rules will need tweaking for outputting custom properties

Reply:
it is [valid Sass](https://sass-lang.com/documentation/modules#rgb) and not valid CSS. previously the output would be converted from hex to rgb values with the Sass `rgba` function (Sass => CSS). after the recent theming work we are emitting custom properties with fallback values (Sass => custom properties => CSS) but the fallbacks are still referencing Sass functions